### ISSUE TITLE
convert uploaded  .PNG file to .JPEG or other accepted file type for analysts--Let Emily confirm it first

### ISSUE THREAD

**Issue by DianeLiu2019** [cds-snc/report-a-cybercrime#1884]
## Summary
 .png files were not included in the original list of file types accepted by the CAFC. the Analyst need screen capture the .png image, save as a .pdf and upload to the CAFC. However, a LOT of report submissions include PNG which are screenshots. We need find a way to convert uploaded  .PNG file to .JPEG  or other accepted file type for analysts . 

Dave Heerema's idea: same package that can do conversion may be able to do .isPNG().  This could be done client side during the phase where we analyze the file size / upload validation?

**Comment by ngosset** [comment]
This one should have closed with #1951
